api/swagger.yml

```diff
@@ -176,28 +176,26 @@ components:
           items:
             $ref: "#/components/schemas/Repository"
 
+    MergeResultSummary:
+      description: "Deprecated: merge summary will be removed"
```

Comment:
I would rather only deprecate all the _fields_ and keep a MergeSummary, even empty.  That will prevent _future_ backwards-compatibility failures -- otherwise when we will want to add a response type, that's another breaking change :-(